### DISCUSSION TITLE
Multiple Models AdminUser / User

### DISCUSSION THREAD

**Discussion by henrikbjorn** [Sorcery/sorcery#334] [Q&A]
Currently looking into Sorcery and it looks great so far. However it seems that it isn't possible to have multiple different models authenticate with Sorcery?

Many have a setup like this:

1. Devise with a User model
2. ActiveAdmin which uses Devise and a AdminUser model

Is it possible to do the same with Sorcery but just replace Devise.

The docs shows STI support, but that is not desired or feasible in most cases. As these two models are totally separate in their scope and domain model.

**Comment by joshbuker** [comment]
This is a major reason for the change to how configuration works in the v1 of Sorcery - to allow different configs for different controllers/use cases (e.g., switching the user model to an admin user model).

You might be able to finagle v0 to do something similar, but it's not explicitly supported like it is for v1.

That said, v1 isn't ready for use yet, and I've really struggled to find the time to work on it. I still plan on completing it, hopefully sooner rather than later, but I can't make any promises on timeline.
